docker/README.md

```diff
@@ -0,0 +1,81 @@
+# Run GreenGuard using Docker
+
+GreenGuard is prepared to be run using [Docker](https://docker.com/).
+
+This are the commands needed to start a Docker container locally that runs a [Jupyter Notebook](
+https://jupyter.org/) already configured to run GreenGuard.
+
+```bash
+docker run -ti -p8888:8888 signals-dev/greenguard:0.2.2.dev0
+```
+
```

Comment:
This will start a Jupyter Notebook instance on your computer already configured to use GreenGuard. You can access it by pointing your browser at http://127.0.0.1:8888